server/__init__.py

```diff
@@ -23,6 +23,9 @@ def create_app(test_config=None):
     from .auth import bp
     app.register_blueprint(bp)
 
+    from .neo4j_interface import Neo4j_Interface
+    neo4j_db = Neo4j_Interface('bolt://localhost:7687', 'neo4j', 'password')
```

Comment:
Q: Do we still need to pass these in as cmd arguments (README.md line 36) if the address, username and password are hardcoded here?